app/models/comment.rb

```diff
@@ -30,14 +30,14 @@ def self.comment_weekly_tallies(span = 52, time = Time.current)
     (0..span).each do |week|
       weeks[span - week] = Comment.select(:timestamp)
         .where(timestamp: time.to_i - week.weeks.to_i..time.to_i - (week - 1).weeks.to_i)
-        .count
+        .size
     end
     weeks
   end
 
   def self.contribution_graph_making(start = Time.now - 1.year, fin = Time.now)
     date_hash = {}
-    week = start.to_date.step(fin.to_date, 7).count
+    week = start.to_date.step(fin.to_date, 7).size
```

Comment:
Hi @Jolios , thanks for opening this :tada: ,,,lets maintain this as `.count` `.size` will return nil here. Thanks

Reply:
@cesswairimu, I committed the change, could you please review it again. Thanks